bpf/lib/nodeport.h

```diff
@@ -2928,7 +2928,7 @@ static __always_inline int nodeport_lb4(struct __ctx_buff *ctx,
 		switch (ret) {
 		case CT_NEW:
 redo:
-			ct_state_new.src_sec_id = WORLD_IPV4_ID;
+			ct_state_new.src_sec_id = src_sec_identity;
 			ct_state_new.node_port = 1;
```

Comment:
Minor patch ordering aspect - let's move this patch a bit further back in the series, after all the "set src identity to world if 0" changes are done?

I'm not aware that any BPF code uses the `src_sec_id` (it's definitely not returned on the CT lookup), did you check how this change would affect the agent/proxy?